bugbug/models/accessibility.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from imblearn.over_sampling import BorderlineSMOTE
+from imblearn.pipeline import Pipeline as ImblearnPipeline
+from sklearn.compose import ColumnTransformer
+from sklearn.feature_extraction import DictVectorizer
+from sklearn.pipeline import Pipeline
+
+from bugbug import bug_features, bugzilla, feature_cleanup, utils
+from bugbug.model import BugModel
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class AccessibilityModel(BugModel):
+    def __init__(self, lemmatization=False):
+        BugModel.__init__(self, lemmatization)
+
+        self.calculate_importance = False
+
+        feature_extractors = [  # TODO: Check effect of adding/removing some features
+            bug_features.HasSTR(),
+            bug_features.Severity(),
+            bug_features.Keywords(),
+            bug_features.Whiteboard(),
+            bug_features.EverAffected(),
```

Comment:
The `EverAffected` feature does not make sense in the context of this model since we want to train the model on the state of the bug at the point of filing. Similarly, for any feature that depends on `history`:

https://github.com/mozilla/bugbug/blob/341c638ee6515bddc80780672c28df4bf63555d3/bugbug/bug_features.py#L311-L312